src/layouts/ReviewRequest/components/SendRequestModal/SendRequestModal.tsx

```diff
@@ -36,11 +36,11 @@ const generateEmailToAdmins = (
   const to = admins[0].value
   const cc = admins.slice(1).map(({ value }) => value)
   const subject = `New review request for ${siteName}`
-  // NOTE: %0D is a newline. Using \n does not work, possibly because
+  // NOTE: %0D%0A is a newline. Using \n does not work, possibly because
   // the body portion is interpreted as text/plain
-  const body = `I’ve asked you to review changes to my site!%0D%0D
+  const body = `I’ve asked you to review changes to my site!%0D%0A%0D%0A
 
-Request title: ${reviewTitle}%0D
+Request title: ${reviewTitle}%0D%0A
```

Comment:
nit: can we define a const NEW_LINE = "%0D%0A" and use this instead?

Reply:
good point, fixed!